src/Umbraco.Web/Models/ContentEditing/MemberSave.cs

```diff
@@ -27,16 +31,27 @@ public class MemberSave : ContentBaseSave<IMember>
         [DataMember(Name = "memberGroups")]
         public IEnumerable<string> Groups { get; set; }
 
-        [DataMember(Name = "comments")]
-        public string Comments { get; set; }
-
-        [DataMember(Name = "isLockedOut")]
-        public bool IsLockedOut { get; set; }
-
-        [DataMember(Name = "isApproved")]
-        public bool IsApproved { get; set; }
-        
-
-        // TODO: Need to add question / answer support
+        /// <summary>
+        /// Returns the value from the Comments property
+        /// </summary>
+        public string Comments => GetPropertyValue<string>(Constants.Conventions.Member.Comments);
```

Comment:
Are it intended the DataMemberAttributes is removed?

Reply:
Yes, these aren't set on the front-end.